scene/gui/graph_edit.cpp

```diff
@@ -260,7 +275,13 @@ void GraphEdit::_scroll_moved(double) {
 	if (!awaiting_scroll_offset_update) {
 		call_deferred(SNAME("_update_scroll_offset"));
 		awaiting_scroll_offset_update = true;
+
+		// In Godot, signals on change value are avoided as a convention.
+		if (!setting_scroll_ofs) {
+			emit_signal(SNAME("scroll_offset_changed"), get_scroll_ofs());
+		}
```

Comment:
Moving it here means that we will emit the signal before the value has been updated, which is incorrect.